bazel_tools/scala.bzl

```diff
@@ -49,6 +49,16 @@ common_scalacopts = [
     "-Xfatal-warnings",
     # catch missing string interpolators
     "-Xlint:missing-interpolator",
+    "-Xlint:by-name-right-associative",  # will never be by-name if used correctly
```

Comment:
This is fixed by scala/scala#5969 for 2.13, so we can remove it if we're 2.13 only.